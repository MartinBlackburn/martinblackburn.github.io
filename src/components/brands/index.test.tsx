// libraries
import { render } from "@testing-library/react";

// component under test
import Brands from "./";

describe("Brands component", () => {
    test("renders correctly", () => {
        const { asFragment } = render(<Brands />);

        expect(asFragment()).toMatchSnapshot();
    });
});
