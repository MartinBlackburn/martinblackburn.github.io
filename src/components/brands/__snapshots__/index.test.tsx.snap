// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Brands component renders correctly 1`] = `
<DocumentFragment>
  <section
    class="brands"
  >
    <h1>
      Brands I've worked with:
    </h1>
    <div
      class="brands__images"
    >
      <img
        alt="bbc logo"
        src="/images/brands/bbc.png"
      />
      <img
        alt="gripple logo"
        src="/images/brands/gripple.png"
      />
      <img
        alt="guardian jobs logo"
        src="/images/brands/guardian jobs.png"
      />
      <img
        alt="lapicida logo"
        src="/images/brands/lapicida.png"
      />
      <img
        alt="national club golfer logo"
        src="/images/brands/national club golfer.png"
      />
      <img
        alt="wren kitchens logo"
        src="/images/brands/new scientist.png"
      />
      <img
        alt="sky betting logo"
        src="/images/brands/sky betting.png"
      />
      <img
        alt="washington post logo"
        src="/images/brands/washington post.png"
      />
      <img
        alt="wren kitchens logo"
        src="/images/brands/wren kitchens.png"
      />
    </div>
  </section>
</DocumentFragment>
`;
